src/shared/lib/GenericAssayUtils/GenericAssayCommonUtils.ts

```diff
@@ -358,3 +358,21 @@ export function deriveDisplayTextFromGenericAssayType(
     }
     return derivedDisplayText;
 }
+
+export function getSortedGenericAssayTabInfo(
+    genericAssayEnrichmentProfilesGroupedByGenericAssayType: {
+        [key: string]: MolecularProfile[];
+    } = {}
+): { genericAssayType: string; linkText: string }[] {
+    return _.sortBy<{ genericAssayType: string; linkText: string }>(
```

Comment:
nested lodash is super hard to read. can we use _.chain().keys().map().value() ?

Reply:
separated this into multiple operations to make it easier to read